### PR TITLE
Added column chooser entry

### PR DESCRIPTION
**Pull request by FinalButterfly**

### CODE REVIEW

File: source/plugin/column-chooser.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Column chooser
+group: Plugins
+order: 15
+---
+- [Installation](#installation)
+
+Column chooser plugin
+
+{% docEditor "github/qgrid/ng2-example/tree/action-bar-basic/latest" %}
```
Comment (klumba12) [inline source/plugin/column-chooser.md:11]: why action-bar basic?

File: source/plugin/column-chooser.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<a name="installation" href="#installation">
+   Installation
+</a>
+
+This plugin is pre-installed in qgrid and will be rendered by default.
+
+To remove the column chooser, empty the actions array.
```
Comment (klumba12) [inline source/plugin/column-chooser.md:19]: please add `how to` and add code how to remove column chooser

File: source/plugin/column-chooser.md (hunk 1 above)
Comment (FinalButterfly) [reply source/plugin/column-chooser.md:11]: Column chooser is present in almost every demo.

File: source/plugin/column-chooser.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+			[model]="gridModel">
+			<q-grid-columns generation="deep">
+			</q-grid-columns>
+		</q-grid>
+	`,
+	providers: [DataService],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ExampleActionBarOnEnterComponent {
+	rows: Observable<Atom[]>;
```
Comment (klumba12) [inline source/plugin/column-chooser.md]:
```typescript
rows$ = this.dataService.getAtoms();
gridModel = this.qgrid
   .model()
   .action({ items: [] } );
```

**Review by klumba12 — APPROVED** [review]

**Pull request merged**